apps/glusterfs/errors.go

```diff
@@ -25,4 +25,8 @@ var (
 	ErrKeyExists        = errors.New("Key already exists in the database")
 	ErrNoReplacement    = errors.New("No Replacement was found for resource requested to be removed")
 	ErrCloneBlockVol    = errors.New("Cloning of block hosting volumes is not supported")
+
+	// well known errors for cluster device source
+	EmptyClusterErr = errors.New("No nodes in cluster")
+	NoStorageErr    = errors.New("No online storage devices in cluster")
```

Comment:
I would prefer to stick with the existing naming scheme:
```
ErrEmptyCluster = ...
ErrNoStorage     = ...
```

Reply:
Oops. Don't know how I missed that.